netlify/pages/about.yaml

```
@@ -167,7 +232,7 @@ collage:
 actionBar:
   ctas:
     - color: accent
-      href: 'https://angel.co/company/stoplight/jobs'
+      href: 'https://stoplight.careerplug.com/jobs'
```

Why are we switching to career plug? IMO angel.co has a much better UI. What kind of customization do we have over that site?

```suggestion
      href: 'https://angel.co/company/stoplight/jobs'
```

---

Not sure who has final say on this. Doesn't matter to me, but I thought we were moving to Career Plug as main site.

---

Let's keep it as angel.co until we have a better solution. Career plug's UI is not up to our standards lol

---

I've added a ticket to create our own solution: https://github.com/stoplightio/stoplight.io/issues/1271